packages/discussions/lib/Notifications.js

```diff
@@ -25,21 +25,21 @@ const {
   DEFAULT_MAIL_FROM_ADDRESS,
   DEFAULT_MAIL_FROM_NAME,
   FRONTEND_BASE_URL,
-  GENERAL_FEEDBACK_DISCUSSION_ID,
 } = process.env
 
-const getDiscussionUrl = async (discussion, context) => {
-  const communityUrl = `${FRONTEND_BASE_URL}/dialog?id=${discussion.id}`
-  if (discussion.id === GENERAL_FEEDBACK_DISCUSSION_ID) {
-    return `${communityUrl}&t=general`
+const getDiscussionPath = (discussion) => {
+  // currently discussion.path is synched with document path on publish
+  // - for template discussion the document path is always suffixed with /diskussion
+  //   in that case we don't want to rewrite the path
+  // - in all other cases, mainly template article,
+  //   we rewrite to /dialog with the discussion id
+  if (discussion.repoId && !discussion.path.match(/\/diskussion$/)) {
```

Comment:
After sleeping on it:
To handle the edge case of template `discussion` being rewritten to a nice path ([example](https://www.republik.ch/brigitte-meyer)):
```suggestion
  if (discussion.repoId && !discussion.path.match(/\/diskussion$/) && discussion.path.match(/^\/[0-9]{4}\/[0-9]{2}\/[0-9]{2}\//)) {
```

Or revert yo the old way: query document and check that it is template `article`. Or DB migration ;-)